web/src/pages/KellyannegeorgePage/KellyannegeorgePage.js

```diff
@@ -0,0 +1,34 @@
+import { Link, routes } from '@redwoodjs/router'
+import { Center } from '@chakra-ui/react'
+import { MetaTags } from '@redwoodjs/web'
+import { Heading , Text} from '@chakra-ui/react'
+
+const KellyannegeorgePage = () => {
+
+  return (
+
+    <body>
```

Comment:
There should not be <body> tags in the jsx code ... be aware that you are not coding an HTML page, this is a Redwood page ... Redwood pages are React components.

Please change the <body></body> to blank placeholder tags: <></>